### DISCUSSION TITLE
[A-L] (2023/24) Foglio 15 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#285] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2ab42697-2472-4ead-9dc7-72aefaab8e92)

**Comment by Elia-Belli** [comment]
$W$ è fornito in forma di equazione cartesiana, per ottenere una base risolviamo il seguente sistema (passiamo all'eq parametrica):

$$\begin{cases}x_1-x_3+x_4+x_5\end{cases}\to (1\ \ 0\ -1\ \ 1\ \ 1)\to \begin{cases}x_1=x_3-x_4-x_5\\
x_2=x_2\\
x_3=x_3\\
x_4=x_4\\
x_5=x_5\end{cases}$$

Ho quindi un sistema con 4 variabili libere e 1 variabile dipendente, dal quale ricavo la base:
$$W=Span\\{(0,1,0,0,0),(1,0,1,0,0),(-1,0,0,1,0),(-1,0,0,0,1)\\}$$

**Comment by CiottoloMaggico** [comment]
Analogo a @Elia-Belli, anche se ho cancellato gli scarabocchi il ragionamento è lo stesso
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/a8468b3b-35b7-4025-aeda-0c7f3390a3c0)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/da67c793-627f-4a2e-a37a-89d777064a63)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-25 163947](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/b6a8e1ce-8004-4729-b93d-02321476fee3)
